cmd/infracost/run.go

```diff
@@ -264,6 +272,30 @@ func runMain(cmd *cobra.Command, runCtx *config.RunContext) error {
 	return nil
 }
 
+func loadInfracostJSONSnapshot(snapshot string) (output.Root, error) {
+	_, err := os.Stat(snapshot)
+	if errors.Is(err, os.ErrNotExist) {
+		return output.Root{}, fmt.Errorf("%s used by --compare-to flag does not exist", snapshot)
+	}
+
+	if err != nil {
+		return output.Root{}, fmt.Errorf("Cmould not load %s used by --compare-to flag, err: %s", snapshot, err)
+	}
+
+	b, err := os.ReadFile(snapshot)
+	if err != nil {
+		return output.Root{}, fmt.Errorf("Cmould not read %s used by --compare-to flag, err: %s", snapshot, err)
```

Comment:
```suggestion
		return output.Root{}, fmt.Errorf("Could not read %s used by --compare-to flag, err: %s", snapshot, err)
```

Reply:
👍

Reply:
resolved in https://github.com/infracost/infracost/pull/1570/commits/56139482c4335aff9a42afb2295487ef1ecefeb5